packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -260,7 +260,8 @@ export const getSelectedAccount = (
     );
 };
 
-export const getNetwork = (symbol: string) => NETWORKS.find(c => c.symbol === symbol) || null;
+export const getNetwork = (symbol: string): Network | null =>
+    NETWORKS.find(c => c.symbol === symbol) || null;
```

Comment:
is verbose return type needed here?

Reply:
yup, typescript started complaining because NETWORKS object got too large after adding additional cardano networks